css/properties/background-position.json

```diff
@@ -50,7 +50,7 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "28"
```

Comment:
I tested manually on BrowserStackLive and found all the keywords to be supported in Chrome 25 with this example:

```html
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style>
      /* Legacy css to support existing live samples */
      body {
        padding: 0;
        margin: 0;
      }

      svg:not(:root) {
        display: block;
      }

      /* Shared among all <div>s */
      div {
        background-color: #ffee99;
        background-repeat: no-repeat;
        width: 100px;
        height: 100px;
        margin-bottom: 12px;
      }

      .example {
        background-image: url("https://896dd0fcd0151c0cc70609e357609c55e2d5f6d2.mdnplay.dev/en-US/docs/Web/CSS/background-position/star-transparent.gif"),
          url("https://896dd0fcd0151c0cc70609e357609c55e2d5f6d2.mdnplay.dev/en-US/docs/Web/CSS/background-position/cat-front.png");
      }

      .bottom {
        background-position: bottom;
      }
      .center {
        background-position: center;
      }
      .left {
        background-position: left;
      }
      .right {
        background-position: right;
      }
      .top {
        background-position: top;
      }
    </style>
  </head>
  <body>
    <div class="example bottom">bottom</div>
    <div class="example center">center</div>
    <div class="example left">left</div>
    <div class="example right">right</div>
    <div class="example top">top</div>
  </body>
</html>
```

Screenshot:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/711682e8-d49f-4efa-be85-16897e5df4dc">

Reply:
Hm, it also seems to have been supported in Chrome 15 (the oldest version testable on BrowserStack Live):

<img width="568" alt="image" src="https://github.com/user-attachments/assets/9c41943c-f162-4507-a7e6-cdd614562b4f">

Reply:
I think we may have another case of the Collector thinking the feature is not supported, because the browser expands the short-hand, so the read value is not equal to the set value:

<img width="300" alt="image" src="https://github.com/user-attachments/assets/e1590f99-0d19-4cd3-b400-35274c884c73">